### DISCUSSION TITLE
Auto Update?

### DISCUSSION THREAD

**Discussion by blogmangit** [bromite/bromite#1524] [Q&A]
is it true that bromite will be automatically updating itself without fdroid from version 94.x.x.109???

**Comment by uazo** [comment]
to be precise, the installation is the responsibility of the user.
from that version it informs the user of the presence of an update and allows you to download the corresponding file.

**Reply by blogmangit** [reply]
Yes man even that is good at least we can get rid of fdroid because fdroid stored the cached files even if we set the time for 1 hour it doesn't delete it. so we have to clear cache manually.

**Comment by csagan5** [comment]
See https://www.bromite.org/news/2021-10-16-bromite-v94-released-with-auto-update-feature
